cg/services/delivery_message/messages/bam_message.py

```diff
@@ -0,0 +1,14 @@
+from cg.services.delivery_message.messages.delivery_message import DeliveryMessage
+from cg.services.delivery_message.messages.utils import get_caesar_delivery_path
+from cg.store.models import Case
+
+
+class BamMessage(DeliveryMessage):
+
+    def create_message(self, cases: list[Case]) -> str:
+        delivery_path: str = get_caesar_delivery_path(cases[0])
+        return (
```

Comment:
Would we want to immediately include the case names in the delivery message?

Reply:
This is a good point. We decided to have one case per sample so we can't assume that all the samples in an order will be delivered at the same time.

Reply:
We ended up naming the cases f"{sample_name}-case" so listing the cases would be a bit ugly. Ideally we should list the sample.name for each sample delivered. What do you guys think?

Reply:
Sample names make sense to me.